### PR TITLE
Decisions of lesson maintainers committee

### PR DESCRIPTION
**Pull request by gvwilson**
One placeholder, others still need editing, one more to come.

### CODE REVIEW

**Comment by wking** [comment]
On Thu, May 05, 2016 at 02:43:40PM -0700, Greg Wilson wrote:
“A subcommittees/maintainers/exercises-and-solutions.md (13)”
“A subcommittees/maintainers/release-process.md (50)”

Links to the discussion behind these decisions would be nice.  I'll
get you some one lists.software-carpentry.org is back up.  And I think
this makes more sense to land in the lesson-template docs (currently
lesson-example) where it's more discoverable and more likely to be
kept up to date.  This repo can always link to the PRs and merge
commits that land the new docs if we want a historical record of
official decisions in this repo.

“A subcommittees/maintainers/handover.md (5)”

Already landed via #77?

“A subcommittees/maintainers/installation-instructions.md (9)”

I'd rather land the (stalled?) swcarpentry/lesson-example#24, which as
far as I can tell is ready to go.

File: subcommittees/maintainers/release-process.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Release Process and Schedule
+
+We have decided to use a **6-month release cycle** for releases, which
+will be named by the year and month they happen, e.g., `2016.05`.
+
+## Overview
+
+1. Each lesson will continue live in the `gh-pages` branch of its own repository.
+2. When a release has to be made, the *lesson maintainer* (or maintainers) will create a branch named after the release, e.g., `2016.05`.
```
Comment (rgaiacs) [inline subcommittees/maintainers/release-process.md]: Why not use tags?

File: subcommittees/maintainers/release-process.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+3. A *release maintainer* will generate HTML pages for that release and add them to the branch.
```
Comment (rgaiacs) [inline subcommittees/maintainers/release-process.md]: Maybe provide the generated HTML as one of the downloads on GitHub's releases list? We can also provide EPUB and PDF.

screen shot 2016-05-15 at 15 01 36

File: subcommittees/maintainers/release-process.md (hunk 1 above)
Comment (gvwilson) [inline subcommittees/maintainers/release-process.md]: I believe this was discussed by the maintainers before the vote.

File: subcommittees/maintainers/release-process.md (hunk 2 above)
Comment (gvwilson) [inline subcommittees/maintainers/release-process.md]: I think that's a separate issue from how we flag particular releases - 
want me to bring it to the maintainers?

File: subcommittees/maintainers/release-process.md (hunk 1 above)
Comment (wking) [inline subcommittees/maintainers/release-process.md]: On Sun, May 15, 2016 at 07:03:06AM -0700, Greg Wilson wrote:
> > +2. When a release has to be made, the *lesson maintainer* (or
> > maintainers) will create a branch named after the release, e.g.,
> > `2016.05`.
> 
> I believe this was discussed by the maintainers before the vote.

I haven't found branch vs. tag discussion in the recent maintainers@
thread, with the closest thing probably being [1] using ‘versiontag’.
The message kicking off the discussion is [2], and the linked
procedure uses branches [3].  There's a later post sumarizing thread
discussion [4], but still no branch vs. tag discussion.  I'll look for
earlier threads to see if I can turn up the branch vs. tag arguments.

I haven't found an actual vote or results yet either ;).  Will post
back once I do.

[1]: http://lists.software-carpentry.org/pipermail/maintainers/2016-April/000230.html
[2]: http://lists.software-carpentry.org/pipermail/maintainers/2016-April/000222.html
[3]: https://github.com/twitwi/swc-meta/blame/1e1c7fc7e02a197f3b939b10a16be1589d068bae/release-process.md#L15
[4]: http://lists.software-carpentry.org/pipermail/maintainers/2016-April/000245.html

File: subcommittees/maintainers/release-process.md (hunk 2 above)
Comment (rgaiacs) [inline subcommittees/maintainers/release-process.md]: > want me to bring it to the maintainers?

Please.

**Comment by rgaiacs** [comment]
@k8hertweck and @gvwilson I think that we can merge this one since we will need to review it in the next few months.

**Comment by k8hertweck** [comment]
Agreed; I can roll this into the standard operating procedures I'm developing for the new maintainers/infrastructure directories and we can reassess if necessary.

**Pull request merged**